sops.go

```diff
@@ -6,6 +6,7 @@ import (
 	"fmt"
 	"go.mozilla.org/sops/kms"
 	"go.mozilla.org/sops/pgp"
+	"go.mozilla.org/yaml.v2"
```

Comment:
We don't have it namespaced under `go.mozilla.org` (and I don't think we should if the end goal is to merge upstream). The correct import path is `github.com/mozilla-services/go-yaml`, I believe.